css/icons.scss

```diff
@@ -8,6 +8,8 @@
 @include icon-black-white('emoji-smile', 'spreed', 1);
 @include icon-black-white('lobby', 'spreed', 1);
 @include icon-black-white('text', 'filetypes', 1, true);
+@include icon-black-white('promoted-view', 'actions', 1, true);
+@include icon-black-white('promoted-view-white', 'actions', 1, true);
```

Comment:
This scss function generate white icons, you do not need both ?
If it's always on top of the main call dark background, shouldn't you be using the white version only?

Reply:
Actually this should be in the `.forced-white {` block couple of lines down, as it will be always white (independent from dark mode) like the video/audio controls.